src/plugins/curlget/README.md

```diff
@@ -4,33 +4,18 @@
 - infos/needs =
 - infos/provides =
 - infos/recommends =
-- infos/placements = pregetstorage
+- infos/placements = getresolver setresolver commit rollback
 - infos/status = unittest configurable readonly preview unfinished nodoc
 - infos/metadata =
 - infos/description = mount remote config files via curl
 
 ## Example ##
 
-    cat /tmp/curltest.ini
-    cat: /tmp/curltest.ini: No such file or directory
+### GET + POST ###
 
-    kdb mount -R noresolver /tmp/curltest.ini system/curltest ini curlget url="https://raw.githubusercontent.com/ElektraInitiative/libelektra/master/src/plugins/ini/ini/plainini"
+kdb mount -R curlget -c get="http://127.0.0.1:8000/curltest.ini",upload="http://127.0.0.1:8000",user="thomas",password="pass",upload/method="POST",upload/postfield="file" /tmp/curltest.ini system/curl ini 
```

Comment:
I think the old example (without upload) is still relevant, please describe them both